packages/web-runtime/tests/components/SidebarQuota.spec.js

```diff
@@ -0,0 +1,55 @@
+import { createLocalVue, mount } from '@vue/test-utils'
+import Vuex from 'vuex'
+import GetTextPlugin from 'vue-gettext'
+import SidebarQuota from 'web-runtime/src/components/SidebarQuota.vue'
+import stubs from '../../../../tests/unit/stubs'
+import DesignSystem from 'owncloud-design-system'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+localVue.use(DesignSystem)
+
+describe('Sidebar Quota component', () => {
+  it.each([2, 3.34, 0])('displays the progressbar', relativeQuota => {
+    const wrapper = mount(SidebarQuota, {
+      store: new Vuex.Store({
+        getters: {
+          quota: function() {
+            return { relative: relativeQuota }
+          }
+        }
+      }),
+      localVue,
+      stubs: { ...stubs, translate: true }
+    })
+    expect(wrapper.find('progress').exists()).toBeTruthy()
+    expect(wrapper.find('progress').element.getAttribute('value')).toEqual(relativeQuota.toString())
+    expect(wrapper.find('progress').element.getAttribute('max')).toEqual('100')
+  })
+  it.each`
```

Comment:
Most of the examples here (following lines) should be examples for a unit test of the `getResourceSize` helper. This unit test here only needs one examples IMO.

Reply:
moved the tests